src/SFA.DAS.AANHub.Api/Controllers/NotificationsController.cs

```diff
@@ -32,4 +35,15 @@ public async Task<IActionResult> Get([FromRoute] Guid id, [FromHeader(Name = Con
 
         return GetResponse(response);
     }
+
+    [HttpPost]
+    [ProducesResponseType(StatusCodes.Status400BadRequest)]
+    [ProducesResponseType(typeof(CreateMemberCommandResponse), StatusCodes.Status201Created)]
+    public async Task<IActionResult> CreateNotification([FromHeader(Name = Constants.RequestHeaders.RequestedByMemberIdHeader)] Guid requestedByMemberId, 
+        [FromBody] CreateNotificationModel model, CancellationToken cancellationToken)
+    {
+        var response = await _mediator.Send(new CreateNotificationCommand() { RequestedByMemberId = requestedByMemberId, MemberId = model.MemberId, NotificationTemplateId = model.NotificationTemplateId }, cancellationToken);
+
+        return GetPostResponse(response, new { id = (response.Result != null) ? response.Result.NotificationId : Guid.NewGuid()});
```

Comment:
You should always have a result here isn't it?

Reply:
What do you mean? This follows the pattern the other POST endpoints in the solution follow to call the GetPostResponse() and send it the value for the route of the GET endpoint it calls (in this case id)